src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime-project.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I assume you still have a goal of removing this and use the core packages correct?